bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -77,6 +77,7 @@ private func dummy() {
     _ = NSLocalizedString("Clear search", comment: "")
     _ = NSLocalizedString("Column %d", comment: "translators: %d: column index.")
     _ = NSLocalizedString("Columns Settings", comment: "")
+    _ = NSLocalizedString("Contact support", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/editor-help/index.native.js#L166:L166)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.